src/components/work/description/Description.js

```diff
@@ -72,11 +72,18 @@ function parseForOccasion(manifestations) {
  * @returns {JSX.Element}
  */
 export function Description({ className = "", data = "", skeleton = false }) {
-  const abstract = data?.abstract?.join(", ");
```

Comment:
this has nothing to do with the content section .. it is a bugfix initiated by afb